3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,87 @@
+
+//Age-ify (A future age calculator)
+
+
+console.log("A future age calculator")
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log(`You will be ${age} years old in ${yearFuture}`);
+
+
+
+
+
+//Goodboy-Oldboy (A dog age calculator)
+
+console.log("age of dog");
+
+const dogYearOfBirth = 2009;
+const dogYearFuture = 2018;
+const humanYears = dogYearFuture - dogYearOfBirth;
+const dogYears = humanYears * 7;
+const shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears) {
+console.log(`Your dog will be ${dogYears} dog years old in ${dogYearFuture}`);} 
+{console.log(`Your dog will be ${humanYears} human years old in ${dogYearFuture}`);}
+
+
+
+
+
+//Housey pricey (A house price estimator)
+
+console.log("house price estimator");
+
+function calculateHousePrice(volumeInMeters, gardenSizeInM2) {
+    return volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+}
+
+// Peter's house
+const peterVolume = 8 * 10 * 10; // width * depth * height
+const peterGardenSize = 100;
+const peterExpectedPriceDKK = calculateHousePrice(peterVolume, peterGardenSize);
+const peterActualPriceDKK = 2500000;
+
```

Comment:
When having values associated with the same entity, in this case, Peter's house we could use an object instead.

For example:

const peterHouse = {
    volume,
    gardenSize,
   ...etc
}